test/unit/puppetlabs/puppetserver/ruby/http_client_test.clj

```diff
@@ -67,6 +70,23 @@
 (def cert-pem (test-resource "localhost_cert.pem"))
 (def privkey-pem (test-resource "localhost_key.pem"))
 
+(schema/defn ^:always-validate
+jruby-config :- jruby-schemas/JRubyConfig
+  "Create a JRubyConfig for testing. The optional map argument `options` may
+  contain a map, which, if present, will be merged into the final JRubyConfig
+  map.  (This function differs from `jruby-tk-config` in that it returns a map
+  that complies with the JRubyConfig schema, which differs slightly from the raw
+  format that would be read from config files on disk.)"
+  ([]
+   (jruby-core/initialize-config
```

Comment:
To cut down on the repetition a bit, seems like this form could be replaced with `(jruby-config {})`.